.github/workflows/pr.yml

```diff
@@ -0,0 +1,19 @@
+name: "Pull Request Checks"
+on:
+  # Allow triggering runs on arbitrary branches
+  workflow_dispatch:
+  pull_request:
```

Comment:
In my experience, the pitfalls are more apparent in a monorepo setup, particularly if you have a workload you want to vary by filetype/specific included files, which, for tests, could be a bunch of not testable/relevant files with the same types as the actual production ones (especially if no additional convention in place to separate them - e.g. folder structure or (additional) bespoke file extension). That doesn't really apply here though, because this isn't a monorepo to begin with, and the tests we're running are (more or less) relevant to the whole application. 

It is possible a new folder/file gets added that's not accounted for, and then the tests run, but that probably won't happen very often (or so I'm guessing), and it can just be added to the ignore list at that time. (the 'waste' aspect is also much worse with larger repos seeing constant updates)

So, I will proceed with putting together an `ignore-paths`-based version of triggers 👍

Reply:
This `paths-ignore` list looks great to me. Thanks!